.github/fixtures/test-release-statistics/cliff.toml

```
@@ -0,0 +1,48 @@
+# git-cliff ~ configuration file
+# https://git-cliff.org/docs/configuration
+
+[changelog]
+# A Tera template to be rendered for each release in the changelog.
+# See https://keats.github.io/tera/docs/#introduction
+body = """
+{% if version %}\
+    ## [{{ version | trim_start_matches(pat="v") }}] - {{ timestamp | date(format="%Y-%m-%d") }}
+{% else %}\
+    ## [unreleased]
+{% endif %}\
+{% for group, commits in commits | group_by(attribute="group") %}
+    ### {{ group | upper_first }}
+    {% for commit in commits %}
+        - {{ commit.message | upper_first }}\
+    {% endfor %}
+{% endfor %}
+### Commit Statistics
+* {{ statistics.commit_count }} commit(s) contributed to the release.
+{%- if statistics.commits_timespan is defined %}
```

These `is defined` checks can be simplified as:

```suggestion
{%- if statistics.commits_timespan %}
```

Also, is there a chance that these are not defined?

I think we can further simplify this template.